ui/lib/kubernetes/addon/components/page/overview.js

```diff
@@ -6,30 +6,21 @@ import { action } from '@ember/object';
 export default class OverviewPageComponent extends Component {
   @service router;
 
-  @tracked roles = [];
   @tracked selectedRole = null;
-  @tracked roleOptions = this.roles.map((role) => {
+  @tracked roleOptions = this.args.model.roles.map((role) => {
```

Comment:
Sorry I didn't notice this the first time around but I think it's probably better to set `roleOptions` in the constructor after calling `super`.

Reply:
No problem! Just fixed!